code/game/mecha/mech_fabricator.dm

```diff
@@ -84,19 +84,16 @@
 	return 1
 
 /obj/machinery/mecha_part_fabricator/emag_act()
-	if(emagged)

```

Comment:
he wants it like
```
if(emagged)
    return
etc...
```
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-early-return